apps/example/priv/repo/migrations/20161123003006_create_thesis_files_table.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Example.Repo.Migrations.CreateThesisFilesTable do
+  @moduledoc false
+  use Ecto.Migration
+
+  def change do
+    create table(:thesis_files) do
+      add :slug, :string
+      add :content_type, :string
+      add :filename, :string
+      add :binary, :binary
```

Comment:
Let's change this field name to `data` rather than `binary`, which means something else in Elixir.